crates/net/eth-wire/src/types/message.rs

```diff
@@ -23,9 +23,11 @@ pub struct ProtocolMessage {
 impl ProtocolMessage {
     /// Create a new ProtocolMessage from a message type and message rlp bytes.
     pub fn decode_message(
-        message_type: EthMessageID,
+        version: EthVersion,
```

Comment:
yeah this won't work if there's other capabilities in the future, we need to separate the validation. i guess it's fine for now.